### PR TITLE
Feature#28 report attendance by day

### PR DESCRIPTION
**Pull request by Dcastanedaan**
# Description

- The post table was deleted
- Changed the employee table fields checkIn and checkOut to check_in and check_out fields

- The checker queries have been modified
- Corrected redirects in the new employee view
- Added functionality to deactivate employees
- Validate that the id_employee field is unique

### CODE REVIEW

File: app/app/controllers/report_controller.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class ReportController < ApplicationController
   def index
-    @reports = DailyLog.where("EXTRACT(MONTH FROM day) = 9").joins("INNER JOIN employees ON employees.status = true").to_a.count -(Employee.where("status = true").to_a.count*Time.days_in_month(Time.current.month))
+  	@active_employees = Employee.where("status = true")    
+    @monthly_logs = DailyLog.where("EXTRACT(MONTH FROM day) = 9").joins(:employee ).where(" employees.status = true ")
+    @reports = @monthly_logs.count -(@active_employees.count*Time.days_in_month(Time.current.month))
+    @employeeday = @monthly_logs.where(" EXTRACT(DAY FROM day) = 14 ").count - @active_employees.count
```
Comment (JuanequeX) [inline app/app/controllers/report_controller.rb:6]: This method can be refactor, separate it to not introduce all this logic in one method.

File: app/app/views/employees/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,14 @@
 <h2> Employee Details </h2> <br>
 <%= link_to 'Back', store_path(params[:store_id]) %>  
 <%= link_to 'Edit', edit_store_employee_path(params[:store_id], params[:id]) %>
+
 <section> 
 <div> Name: <%= @employee.name %> <div/>
 <div> Email: <%= @employee.email %> <div/>
 <div> Private number: <%= @employee.private_number %> <div/>
 <div> Position: <%= @employee.position %> <div/>
 <div> Id number: <%= @employee.id_employee %> <div/>
```
Comment (JuanequeX) [inline app/app/views/employees/show.html.erb:10]: All this `div` label are all wrong closing, please fix it like this way -> `</div>`

File: app/app/views/employees/show.html.erb [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+<div> Status: <%= @employee.status %></div>
 <section/>
```
Comment (JuanequeX) [inline app/app/views/employees/show.html.erb:12]: same this here, [like this](https://github.com/BC-MAY-21-ROR/kata-06-el-reloj-checador-k6-ca-reilero-02/pull/32/files#:~:text=All%20this%20div%20label%20are%20all%20wrong%20closing%2C%20please%20fix%20it%20like%20this%20way%20%2D%3E%20%3C/div%3E)

File: app/db/seeds.rb [hunk 4]
```diff
@@ -17,7 +17,7 @@
  10.times do
   name=Faker::Name.name_with_middle
   emp=Employee.create(name: name, email: Faker::Internet.email(name: name.split(' ')[0]), position: Faker::Job.position, id_employee: Faker::Number.number(digits: 5), private_number: Faker::Number.number(digits: 5), store: company)
-  DailyLog.create!(day: Date.current, checkIn:'05:30:00', checkOut:'10:30:00',employee:emp)
+  DailyLog.create!(day: Date.current, check_in:'05:30:00', check_out:'10:30:00',employee:emp)
```
Comment (JuanequeX) [inline app/db/seeds.rb:20]: indented correctly this line.

**Review by JuanequeX — DISMISSED** [review]

File: app/app/views/employees/show.html.erb (hunk 2 above)
Comment (willcab) [reply app/app/views/employees/show.html.erb]: resolve 👍

File: app/app/controllers/report_controller.rb (hunk 1 above)
Comment (willcab) [reply app/app/controllers/report_controller.rb]: resolve 👍

**Review by JorgeRodz — APPROVED** [review]
LGTM

**Review by CalixtoLopez — APPROVED** [review]
Good job

**Pull request merged**